schemas/config_sh_header.templ

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#================
+# FILE          : config.sh
+#----------------
+# PROJECT       : SUSE Public Cloud recipes
+# COPYRIGHT     : (c) 2021 SUSE LLC. All rights reserved
```

Comment:
The year should be a variable so we do not have to touch the template every year to update.